packages/shared/src/components/cards/v1/WelcomePostCard.tsx

```diff
@@ -66,35 +80,58 @@ export const WelcomePostCard = forwardRef(function SharePostCard(
         }
       }
     >
-      <PostCardHeader
-        post={post}
-        onMenuClick={(event) => onMenuClick?.(event, post)}
-        metadata={{
-          topLabel: enableSourceHeader ? post.source.name : post.author.name,
-          bottomLabel: enableSourceHeader
-            ? post.author.name
-            : `@${post.source.handle ?? post.sharedPost.source.handle}`,
-        }}
-      >
-        <SquadHeaderPicture
-          author={post.author}
-          source={post.source}
-          reverse={!enableSourceHeader}
-        />
-      </PostCardHeader>
-      <CardTitle
-        className={classNames(
-          generateTitleClamp({
-            hasImage: !!image,
-            hasHtmlContent: !!post.contentHtml,
-          }),
-          'multi-truncate',
-        )}
-      >
-        {post.title}
-      </CardTitle>
+      <CardContainer>
+        <PostCardHeader
+          post={post}
+          onMenuClick={(event) => onMenuClick?.(event, post)}
+          metadata={{
+            topLabel: enableSourceHeader ? post.source.name : post.author.name,
+            bottomLabel: enableSourceHeader
+              ? post.author.name
+              : `@${post.source.handle ?? post.sharedPost.source.handle}`,
+          }}
+        >
+          <SquadHeaderPicture
+            author={post.author}
+            source={post.source}
+            reverse={!enableSourceHeader}
+          />
+        </PostCardHeader>
+
+        <CardContent>
+          <div className="mr-4 flex-1">
+            <CardTitle
+              className={classNames(
+                generateTitleClamp({
+                  hasImage: !!image,
+                  hasHtmlContent: !!post.contentHtml,
+                }),
+                'multi-truncate',
+              )}
+            >
+              {post.title}
+            </CardTitle>
+
+            {content && (
+              <p className="my-4 line-clamp-6 break-words typo-callout">
+                {decodedText}
```

Comment:
I believe we should be using this here as well `const { title, summary } = useTruncatedSummary(post);`

Reply:
the decoded text will be a bit trickier because it is potentially HTML, and we just truncated it based on length we run the risk of getting broken HTML. I suggest we merge this as is (so we can publish and start doing testing) while we tackle this particular task separately?

Reply:
So the decoded text is not just text? Just concerned if we don't add this it might mess with designs and requirements...
Still html or not, if we're getting the innerText that should have characters, and we would be able to check the length no?

Reply:
this is built based on `post.contentHtml` which comes in the following format: 
```
<p>Welcome to test, a dedicated space to collaborate, share knowledge, and discuss topics that matter to us!</p>
<p>Here are some of the things you can do in Squads:</p>
<ul>
<li>Say hi: Start by saying hi in the comments below so that we’ll know you’re here</li>
<li>Create a new post: Share interesting links and your thoughts by creating a new post in the squad</li>
<li>Interact with others: Comment and upvote on other members’ posts and give feedback</li>
<li>Personalize it: Customize your profile by adding a profile picture, bio, and links to your projects or social media</li>
<li>Invite other developers you know and appreciate that you think can benefit from this squad</li>
</ul>
<p>Now that you know what you can do in this squad, we’ve put together a code of conduct that we expect all of our squad members to follow:</p>
<ol>
<li>Keep it relevant: Keep your posts and comments relevant to the topic of the Squad. Please refrain from spamming or promoting unrelated content.</li>
<li>Be respectful: Treat others the way you want to be treated. We do not tolerate hate speech, discrimination, or harassment of any kind.</li>
<li>Be constructive: Offer helpful feedback and constructive criticism rather than tearing others down.</li>
<li>Protect your privacy: Do not share personal information or sensitive data in Squads.</li>
</ol>
<p>We hope you will find test useful!</p>
```

This is why truncating this is going to be a bit more complex. We have the `line-clamp-6` as a stopgap to ensure this does not overflow out of control, but I propose we tackle this separately if we want to really enforce a 300 character limit on this.

Reply:
Cool then if we have the `line-clamp-6` I think we should be fine for this not to overflow. Maybe would be good to just note this in the channel and tag design that we're moving on like this for the welcome card because of this possible issue. @DragosIorgulescu would u mind sending a message?